README.md
![alt XOOPS CMS](https://xoops.org/images/logoXoops4GithubRepository.png)
## Xoosla Core module for [XOOPS CMS 2.5.8+](https://xoops.org)
[![XOOPS CMS Module](https://img.shields.io/badge/XOOPS%20CMS-Module-blue.svg)](https://xoops.org)
[![Software License](https://img.shields.io/badge/license-GPL-brightgreen.svg?style=flat)](http://www.gnu.org/licenses/gpl-2.0.html)

[![Scrutinizer Code Quality](https://img.shields.io/scrutinizer/g/mambax7/xooslacore.svg?style=flat)](https://scrutinizer-ci.com/g/mambax7/xooslacore/?branch=master)
[![Codacy Badge](https://api.codacy.com/project/badge/grade/2d27c0023ee54f0b9ba2b5d17a68b2a5)](https://www.codacy.com/app/mambax7/xooslacore)
[![Code Climate](https://img.shields.io/codeclimate/github/mambax7/xooslacore.svg?style=flat)](https://codeclimate.com/github/mambax7/xooslacore)
[![SensioLabsInsight](https://insight.sensiolabs.com/projects/9dc918fe-ea63-4675-832c-8f6c74cdf78f/mini.png)](https://insight.sensiolabs.com/projects/9dc918fe-ea63-4675-832c-8f6c74cdf78f)
[![Latest Pre-Release](https://img.shields.io/github/tag/XoopsModules25x/xooslacore.svg?style=flat)](https://github.com/XoopsModules25x/xooslacore/tags/)
[![Latest Version](https://img.shields.io/github/release/XoopsModules25x/xooslacore.svg?style=flat)](https://github.com/XoopsModules25x/xooslacore/releases/)

**XooslaCore module** for [XOOPS CMS](https://xoops.org) is a total rewrite of the WF-Resources module, it is smaller, 
faster will help in the aid of Xoosla Module development in the future and providing many new features both for 
(Xoosla) developers and users alike. Automatic updates, stats on Xoosla modules and more.

[![Tutorial Available](https://xoops.org/images/tutorial-available-blue.svg)](https://www.gitbook.com/book/xoops/xooslacore-tutorial/) Tutorial: see [GitBook](https://www.gitbook.com/book/xoops/xooslacore-tutorial/). 
To contribute to the Tutorial, [fork it on GitHub](https://github.com/XoopsDocs/xooslacore-tutorial)

[![Translations on Transifex](https://xoops.org/images/translations-transifex-blue.svg)](https://www.transifex.com/xoops) 

Please visit us on https://xoops.org

**Current and upcoming "next generation" versions of XOOPS CMS are being crafted on GitHub at: https://github.com/XOOPS**



docs/changelog.txt
<h5>1.00 Beta 1  [NOT RELEASED]</h5>  Dev: Xoops 2.5.9, PHP 7.2.3
<hr>
- XOOPS 2.5.9, PHP 7 (mamba)
- removed @version (mamba)
- PSR-2 code cosmetics (mamba)
- addNavigation(basename(__FILE__))  (mamba)
- dirname(__FILE__) to __DIR__ (mamba)
- replaced < br /> with < br > (mamba)
- Unnecessary double quotes (mamba)
- reference mismatch (mamba)
- Hardening to type safe (mamba)
- changed _handler to Handler (mamba)
- added 'true' for tokens in forms (mamba)
- renamed .html Smarty templates to .tpl (mamba)
- removed '/' from void HTML elements (&lt;br>, &lt;img>, etc) (mamba)
- HTML 5: removed 'selected' and 'checked' (mamba)
- Updates for PayPal changes (zyspec)
- added Contributing.md (mamba)
- added namespaces class autoload (mamba)



<h5>1.00 Alpha 1  [2010-07-05]</h5>
<hr>
- Initial release (Catzwolf)


xoops_version.php
<?php
/**
 * Name: Xoosla Core Module
 * Description:
 *
 * @package    : Xoosla Modules
 * @Module     : Xoosla Core Module
 * @subpackage :
 * @since      : v1.00
 * @author     John Neill <catzwolf@xoosla.com>
 * @copyright  : Copyright (C) 2010 Xoosla Modules. All rights reserved.
 * @copyright    XOOPS Project https://xoops.org/
 * @license      GNU GPL 2 or later (http://www.gnu.org/licenses/gpl-2.0.html)
 */
defined('XOOPS_ROOT_PATH') || die('Restricted access');

require_once __DIR__ . '/preloads/autoloader.php';

$moduleDirName = basename(__DIR__);

/**
 * Module Details
 */
$modversion['version']             = 1.00;
$modversion['requires']            = 1.00;
$modversion['module_status']       = 'Beta 1';
$modversion['release_date']        = '2017/05/14';
$modversion['name']                = XL_MI_XOOSLACORE;
$modversion['description']         = XL_MI_XOOSLACORE_DSC;
$modversion['author']              = XL_MI_XOOSLA_AUTHOR;
$modversion['credits']             = '';
$modversion['status']              = XL_MI_XOOSLA_STATUS;
$modversion['lead']                = XL_MI_XOOSLA_LEAD;
$modversion['contributors']        = XL_MI_XOOSLA_CONTRIBUTORS;
$modversion['website_url']         = 'https://xoops.org';
$modversion['website_name']        = 'XOOPS Project';
$modversion['email']               = '';
$modversion['demo_site_url']       = '';
$modversion['demo_site_name']      = '';
$modversion['support_site_url']    = '';
$modversion['support_site_name']   = '';
$modversion['submit_bug_url']      = '';
$modversion['submit_bug_name']     = '';
$modversion['submit_feature_url']  = '';
$modversion['submit_feature_name'] = '';
$modversion['disclaimer']          = '';
$modversion['license']             = '';
$modversion['official']            = 0;
$modversion['image']               = 'assets/images/logoModule.png';
$modversion['dirname']             = $moduleDirName;

/**
 * SQL
 */
// $modversion['sqlfile']['mysql'] = 'media/sql/mysql.sql';
/**
 * Mysql Tables
 */
// $modversion['tables'][] = '';
/**
 * Admin things
 */
$modversion['hasAdmin']   = 0;
//$modversion['adminindex'] = 'admin/index.php';
//$modversion['adminmenu']  = 'admin/menu.php';

/**
 * Additionnal script executed during install update
 */
// $modversion['onInstall'] = 'include/oninstall.php';
// $modversion['onUpdate'] = 'include/onupdate.php';
// $modversion['onUninstall'] = 'include/onuninstall.php';
/**
 * Frontend
 */
// $modversion['hasMain'] = 0;
/**
 * Frontend
 */
// $modversion['hasSearch'] = 0;
/**
 * Comments
 */
// $modversion['hasComments'] = 0;
/**
 * Notifications
 */
// $modversion['hasNotification'] = 0;
/**
 * Blocks
 */
// $modversion['blocks'][] = array( 'file' => '',
// 'name' => '',
// 'description' => '',
// 'show_func' => 'textbox',
// 'edit_func' => 'text',
// 'options' => '',
// 'template' => ''
// );
/**
 * Templates
 */
// $modversion['templates'][] = array( 'file' => '',
// 'description' => '' );
/**
 * Module Configuration
 */
// $modversion['config'][] = array( 'name' => '',
// 'title' => '',
// 'description' => '',
// 'formtype' => 'textbox',
// 'valuetype' => 'text',
// 'default' => ''
// );
